dbus-macros/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "dbus-macros"
+version = "0.1.0"
+authors = ["David Henningsson <diwic@ubuntu.com>"]
```

Comment:
Please use your own name here. I'll add mine later if I end up make a lot of contributions to it. You deserve to have the credit :-)